TASVideos.WikiEngine/MakeBracketed.cs

```diff
@@ -12,7 +12,7 @@ public static partial class Builtins
 	/// <summary>
 	/// Turns text inside [square brackets] into the appropriate thing, usually module or link.  Does not handle [if:].
 	/// </summary>
-	public static IEnumerable<INode> MakeBracketed(int charStart, int charEnd, string text)
+	public static IEnumerable<INode> MakeBracketed(int charStart, int charEnd, bool isUGC, string text)
```

Comment:
What is UGC?

Reply:
User-generated content https://developers.google.com/search/docs/crawling-indexing/qualify-outbound-links#ugc

Reply:
I'm struggling to understand the code here.  How does it determine which Wiki pages are UGC and which aren't?  Shouldn't we consider ALL wiki pages as UGC? It's a wiki.

Reply:
For wiki-lang content, I've done `IsUGC => wikiPage.IsHomePageOrSubmissionNotes();`